test/Kestrel.Transport.FunctionalTests/ChromeTests.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
```

Comment:
You might want to move this to Kestrel.Transport.Sockets.FunctionalTests so it doesn't get run for the libuv transport functional tests now that #3091 has been merged.